synapse/cortex.py

```diff
@@ -87,6 +87,65 @@ def getNodesBy(self, full, valu, cmpr='='):
     async def fini(self):
         pass
 
+    def addNodeTag(self, iden, tag, valu=(None, None)):
+        '''
+        Add a tag to a node specified by iden.
+
+        Args:
+            iden (str): A hex encoded node BUID.
```

Comment:
How are users supposed to get the buid? calculating it themselves from the ndef?

Reply:
nvm - i see you added the iden to the packed node